modules/nw-metalLB-basic-upgrade-operator.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-upgrading-operator.adoc
+
+:_content-type: PROCEDURE
+
+[id="upgrading-metallb-operator-to-minor-version_{context}"]
+= Upgrading MetalLB Operator to the next minor version
+
+You can manually upgrade MetalLB Operator to the next minor version by using the {product-title} web console to change the channel of your Operator subscription.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Access the web console and navigate to *Operators* -> *Installed Operators*.
+
+. Click *MetalLB Operator* to open the *Operator details* page.
+
+. Update the *Update approval* strategy.
++
+[NOTE]
+====
+Updating the *Update approval* strategy is required because the automatic update is not working as expected.
```

Comment:
Not sure what section is about.
If it's to upgrade from a previous (as in, <4.11 version) this note makes sense, otherwise it does not.
Also, if it's for <4.11, we must ensure users will use the same namespace metallb was previously installed to.

Reply:
This is still open

Reply:
To be on the safe side so that operator does not upgrade automatically on upgrading ocp cluster and end up with bad installation it is good idea to set up install plan manual.

Reply:
Right, what I am unsure of is the `Upgrading MetalLB Operator to the next minor version` part, which seems to relate to the next 4.11z versions